tests/unit/api-transcription-routes.test.ts

```
@@ -0,0 +1,358 @@
+        // Mock total count query
+        const mockFromQuery = {
+          select: vi.fn().mockReturnThis(),
+          eq: vi.fn().mockReturnThis(),
+          lt: vi.fn().mockReturnThis(),
+        }
+        
+        // First call for total count
+        mockFromQuery.eq.mockReturnValueOnce({
+          ...mockFromQuery,
+          // Return 150 total items
+          then: (callback: any) => callback({ data: new Array(150), error: null })
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)